desktop/assets/analytics.coffee

```diff
@@ -12,8 +12,14 @@ require '../analytics/main_layout.js'
 require '../analytics/before_ready.js'
 
 $ -> analytics.ready ->
+  $.ajax '//munchkin.marketo.net/munchkin.js',
+      dataType: 'script'
+      cache: true
+      error: (jqXHR, textStatus, errorThrown) -> $('body').append "AJAX Error: #{textStatus}"
+      success: (data, textStatus, jqXHR) -> $('body').append "Successful AJAX call: #{data}"
```

Comment:
We shouldn't actually render to the `body` here, which the user can see. We can probably remove these `error` and `success` callbacks. Or if you'd prefer for debugging purposes, then use `console.log` instead of `$('body').append`.